jobs/clean_cqc_location_data.py

```
@@ -217,6 +222,58 @@ def split_dataframe_into_registered_and_deregistered_rows(
     return registered_df, deregistered_df
 
 
+def check_current_against_contemporary_geographies(
```

This function is to identify if the CQC postcode provided can be found in the ONS postcode file. If it is, a value will be provided. If not it will be a null entry. If it is null we want to raise an error

on that basis maybe we rename the function to be something like

def raise_error_if_postcode_not_found_in_ons_dataset():

---

Happy to do this